hyperspy/_signals/eds.py

```diff
@@ -191,13 +193,88 @@ def rebin(self, new_shape, out=None):
         this_md = self.metadata
         that_md = s.metadata
         keys = ("Acquisition_instrument.SEM.Detector.EDS.live_time",
-                "Acquisition_instrument.TEM.Detector.EDS.live_time")
+                "Acquisition_instrument.TEM.Detector.EDS.live_time",
+                "Acquisition_instrument.SEM.Detector.EDS.real_time",
+                "Acquisition_instrument.TEM.Detector.EDS.real_time",)
         for key in keys:
             if key in this_md:
                 that_md.set_item(key, this_md.get_item(key) * np.prod(factors))
         return s
     sum.__doc__ = Signal1D.sum.__doc__
 
+    def linear_bin(self, scale, crop=True):
+        """
+        Binning of the spectrum image by a non-integer pixel value.
+
+        Parameters
+        ----------
+        scale : a list of floats
+            for each dimension specify the new:old pixel ratio
+            e.g. a ratio of 1 is no binning
+                 a ratio of 2 means that each pixel in the new spectrum is
+                 twice the size of the pixels in the old spectrum.
+        crop_str : {'False'}, optional
+            when binning by a non-integer number of pixels it is likely that
+             the final row in each dimension contains less than the full quota to
+             fill one pixel.
+             e.g. 5*5 array binned by 2.1 will produce two rows containing 2.1
+             pixels and one row containing only 0.8 pixels worth. Selection of
+             crop_str = 'True' or crop = 'False' determines whether or not this
+             'black' line is cropped from the final binned array or not.
+
+            *Please note that if crop=False is used, the final row in each
+            dimension may appear black, if a fractional number of pixels are left
+            over. It can be removed but has been left to preserve total counts
+            before and after binning.*
+
+        Returns
+        ------
+        s : Signal subclass
+
```

Comment:
The ---- after the "headline" needs to be as many symbols as the headline, so add another - here.